tests/spec/combine_series/test_event_series_and_event_series.py

```diff
@@ -23,3 +23,17 @@ def test_event_series_and_event_series(spec_test):
             2: (201 + 211) + (202 + 212),
         },
     )
+
+
+def test_event_series_and_sorted_event_series(spec_test):
+    """
+    The sort ordering of the underlying event series does not affect their combination.
+    """
+    spec_test(
+        table_data,
+        (e.i1 + e.sort_by(e.s1).i2).minimum_for_patient(),
+        {
+            1: (101 + 111),
+            2: (201 + 211),
+        },
+    )
```

Comment:
This test reassured me that the sorting doesn't affect the outcome of the series combination (and also that the other engines can already deal with it) - but I'm not sure if it makes sense to be included in the docs

Reply:
I had a go at what it would look like to move this test out of the specs tests and into the query engine integration tests and there ends up being quite a lot of boilerplate. I agree that it doesn't make a lot of sense in the docs, but I'd say we just leave it in here for now and we can decide if have to move it out later. It does feel like a useful property to test.